docs/translations/api-docs/data-grid/data-grid-premium/data-grid-premium.json

```diff
@@ -42,7 +42,9 @@
     "columnHeaderHeight": {
       "description": "Sets the height in pixel of the column headers in the Data Grid."
     },
-    "columns": { "description": "Set of columns of type <a href='/x/api/data-grid/grid-col-def/'>GridColDef</a>[]." },
```

Comment:
Did prettier format this?
Should we exclude autogenerated files from prettier?

Reply:
Nevermind, I just saw [`c62a6fb` (#11303)](https://github.com/mui/mui-x/pull/11303/commits/c62a6fbe9bcb3246227fd1b71eecdb733f3d41ec)